### ISSUE TITLE
NVDA's help mode does not work when using scroll backward and forward on braille displays

### ISSUE THREAD

**Issue by MichelSuch** [nvaccess/nvda#10141]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.

<details>
<summary>Details</summary>

</details>

Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
			- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
1. Press NVDA+1 to turn on NVDA's input help mode.
2. On your braille display use scrollback or scrollforward key.

### Actual behavior:
The terminal scrols backward or forward as if help mode was turned off.

### Expected behavior:
NVDA should announce the gesture and the action that should be performed.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2019.2

#### Windows version:
Windows 10 1903 64 bits

**Comment by XLTechie** [comment]
I know nothing about this, but to help people trying to debug this issue, you 
should mention which braille display you are using, in case it is a problem with 
a specific driver. Also: have you tried more than one braille display?

**Comment by Brian1Gaff** [comment]
Many would not have access to one never mind more than one!
Big silly grin...

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by MichelSuch** [comment]
I have noticed the problem with 2 displays:
a Papenmeier Braillex old model and a Seika 40.

**Comment by Oreonan2** [comment]
I also tested with a Brailliant BI 40

**Comment by pzajda** [comment]
I also noticed this behavior with BRLTTY.

**Comment by surfer0627** [comment]
Issue [#3301](https://github.com/nvaccess/nvda/issues/3301): 

> Comment by jteh on 2013-06-24 02:37
The scroll left/right commands always scroll the display, even in input help mode. This is because the input help text might not fit on the display, so the user needs to be able to scroll to read it. I understand the concern here, but I don't know of a way to solve this without making it impossible for braille users to read long input help messages.

**Comment by leonardder** [comment]
Indeed. Here, the ability for braille users to read a complete input help message is more important than input help feedback for these keys.

Closing as duplicate of #3301
